runtime/freelist.c

```diff
@@ -702,7 +702,7 @@ static void ff_reset (void)
 }
 
 /* Note: the [limit] parameter is unused because we merge blocks one by one. */
-header_t *ff_merge_block (value bp, char *limit)
+header_t *caml_ff_merge_block (value bp, char *limit)
```

Comment:
This is one possible fix.  What about making this function `static`?  Does it really need to be exported?

Reply:
Taking a better looks at the file, the other allocation functions in this file are static. Making this function static is thus probably the best option.

Reply:
Indeed, it should be static.